common/config/rush/common-versions.json

```diff
@@ -0,0 +1,43 @@
+/**
+ * This configuration file specifies NPM dependency version selections that affect all projects
+ * in a Rush repo.  For full documentation, please see https://rushjs.io
+ */
+{
+  "$schema": "https://developer.microsoft.com/json-schemas/rush/v5/common-versions.schema.json",
+
+  /**
+   * A table that specifies a "preferred version" for a dependency package. The "preferred version"
+   * is typically used to hold an indirect dependency back to a specific version, however generally
+   * it can be any SemVer range specifier (e.g. "~1.2.3"), and it will narrow any (compatible)
+   * SemVer range specifier.  See the Rush documentation for details about this feature.
```

Comment:
link?

Reply:
There's a link at the top of the file.

---
In reply to: [217225587](https://github.com/Microsoft/rush-example/pull/2#discussion_r217225587) [](ancestors = 217225587)